autoload/neoformat/formatters/pandoc.vim

```diff
@@ -0,0 +1,14 @@
+function! neoformat#formatters#pandoc#enabled() abort
+   return ['pandoc']
+endfunction
+
+function! neoformat#formatters#pandoc#remark() abort
```

Comment:
This should say `...#pandoc#pandoc()` instead of `...#pandoc#remark()`.

The format is `neoformat#formatters#filetype#formatter_name_used_in_enable()`